crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -44,6 +46,11 @@ use crate::types::{
 use super::source_context::get_context_lines;
 use super::SymbolicationActor;
 
+lazy_static! {
```

Comment:
We should use `once_cell` for this, as it is practically the thing that will be stabilized in `std` pretty soon.

Reply:
Ah, I was wondering if we should switch. Will change.